### PR TITLE
Update/add license type to gsuite display on email page

### PR DESCRIPTION
**Pull request by olaseni**
#### Changes proposed in this Pull Request

1. This change modifies the G Suite domains/email list page to display the license type associated with the email user.

2. The emails are also emboldened as a result. This is achieved by adding content to the CSS file accompaniment of the JSX.

To see how this plays out, go to url http://calypso.localhost:3000/email/`domain-slug`, by navigating to [ My Site ] -> [ Manage ] -> [ Domains ] -> [ Email ].

This screen should look like this:

calypso-pr-34314

This patch requires D29818-code

### CODE REVIEW

File: client/my-sites/email/email-management/gsuite-users-card/index.jsx [hunk 1]
```diff
@@ -63,17 +63,18 @@ class GSuiteUsersCard extends React.Component {
 	};
 
 	renderDomain( domain, users ) {
+		const license = users[ 0 ].product_name;
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/gsuite-users-card/index.jsx]: I'm wondering whether a comment explaining that the product name is going to be the same for each user would be useful here. Probably not that much I guess.

File: client/my-sites/email/email-management/gsuite-users-card/index.jsx [hunk 2]
```diff
@@ -158,7 +159,7 @@ const addGoogleAppsUserClick = domainName =>
 	composeAnalytics(
 		recordGoogleEvent(
 			'Domain Management',
-			'Clicked "Add G Suite User" Button in G Suite',
+			'Clicked "Add New User" Button in G Suite',
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/gsuite-users-card/index.jsx]: Good catch.

File: client/my-sites/email/email-management/gsuite-users-card/style.scss [hunk 3]
```diff
@@ -25,6 +25,10 @@
 		&:last-of-type {
 			border-bottom: 1px solid var( --color-neutral-0 );
 		}
+
+		& .gsuite-user-item__email {
+			font-family: $serif;
+		}
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/gsuite-users-card/style.scss]: Why did you add this rule here instead of [here](https://github.com/Automattic/wp-calypso/blob/bee6b9c3a923fc54f45bed959c25c1f896c8fc42/client/my-sites/email/email-management/gsuite-user-item/style.scss#L10)?

File: client/my-sites/email/email-management/gsuite-user-item/style.scss [hunk 4]
```diff
@@ -10,6 +10,7 @@
 	.gsuite-user-item__email {
 		float: left;
 		line-height: 1.7;
+		font-weight: 600;
 	}
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/gsuite-user-item/style.scss]: I know it's not part of coding conventions, but I find rules to be easier to scan when they are kept ordered alphabetically.

File: client/my-sites/email/email-management/gsuite-users-card/index.jsx (hunk 1 above)
Comment (olaseni) [reply client/my-sites/email/email-management/gsuite-users-card/index.jsx]: Certainly makes sense

File: client/my-sites/email/email-management/gsuite-users-card/style.scss (hunk 3 above)
Comment (olaseni) [reply client/my-sites/email/email-management/gsuite-users-card/style.scss]: I just realized now that there is an inner unit for the `GSuiteUserItem` component. Fixing ...
